presto-release-tools/src/main/java/com/facebook/presto/release/tasks/FinalizeReleaseCommand.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.release.tasks;
+
+import com.facebook.presto.release.ForPresto;
+import com.facebook.presto.release.git.GitModule;
+import com.facebook.presto.release.git.GitRepositoryModule;
+import com.facebook.presto.release.maven.MavenModule;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.airline.Command;
+import io.airlift.airline.Option;
+
+import java.util.List;
+
+@Command(name = "finalize-release", description = "Finalize presto release")
+public class FinalizeReleaseCommand
+        extends AbstractReleaseCommand
+{
+    @Option(name = "--upstream-name", title = "Presto repo upstream name")
+    @ConfigProperty("presto.git.upstream-name")
+    public String gitUpstreamName;
```

Comment:
Can these common config options be pulled up into a superclass?

Reply:
Not sure what a proper common superclass should be for `CutReleaseCommand` and `FinalizeReleaseCommand`. I think maybe a little bit of duplicating code is fine, as both class just happens to have the same set of options?

Reply:
They already have a common one, `AbstractReleaseCommand`. They both interact with git in similar ways resulting in the common options. This just seems like low hanging fruit for DRY.